README.md

```diff
@@ -38,6 +38,8 @@ cds compile <path to srv folder> --to ord [-o] [destinationFilePath]
 1) Run `cds watch` in the application's root.
 2) Check the following relative paths for ORD information - `/.well-known/open-resource-discovery` , `/open-resource-discovery/v1/documents/1`.
 
+![](ord.gif)
+
```

Comment:
- file size is too big and should be compressed as it takes time to load
- gif should be sped up as it is very slow

Reply:
have increased the speed